docs/migration-v8.md

```diff
@@ -27,66 +27,67 @@ This section describes how to migrate from execution policies (i.e. `IAsyncPolic
 
 In earlier versions, Polly exposed various interfaces to execute user code:
 
-- `IAsyncPolicy`
-- `IAsyncPolicy<T>`
 - `ISyncPolicy`
+- `IAsyncPolicy`
 - `ISyncPolicy<T>`
+- `IAsyncPolicy<T>`
```

Comment:
Personally I prefer these A-Z sorted.

Reply:
I would say yes unless there's a good reason they're however they are now. Can you point to the other place in the docs you're referring to exactly?

Reply:
- [V7's Timeout](https://github.com/App-vNext/Polly/blob/main/src/Snippets/Docs/Migration.Timeout.cs)
- [V7's RateLimiter](https://github.com/App-vNext/Polly/blob/main/src/Snippets/Docs/Migration.RateLimiter.cs)
- [V7's Bulkhead](https://github.com/App-vNext/Polly/blob/main/src/Snippets/Docs/Migration.Bulkhead.cs)
- [V7's Execute](https://github.com/App-vNext/Polly/blob/main/src/Snippets/Docs/Migration.Execute.cs#L11)
- just to name a few :)

Reply:
Ah, I thought you meant the documentation, not the code.

Reply:
The documentation lists these interfaces only here. Alright, I'll reorder these then.